mmwtex.h

```diff
@@ -88,7 +88,7 @@ void printTexHeader(flag texHeaderFlag);
 flag printTexComment(vstring commentPtr,    /* Sends result to g_texFilePtr */
     flag htmlCenterFlag, /* 1 = htmlCenterFlag */
     long actionBits, /* see indicators below */
-    flag noFileCheck /* 1 = noFileCheck */);
+    flag fileCheck /* 1 = noFileCheck */);

```

Comment:
suspicious comment: parameter name does not match semantics in comment

Reply:
```suggestion
    flag fileCheck /* 1 = fileCheck */);
```

Reply:
ack